packages/postcss-discard-empty/test/index.js

```diff
@@ -85,6 +85,11 @@ test(
   passthroughCSS('h1{/*comment*/}')
 );
 
+test(
+  'should preserve empty custom properties',
+  passthroughCSS('*{--tw-shadow:;}')
```

Comment:
It should be ` ` (empty space), chrome has bug with it, anyway empty value is valid too

Reply:
postcss-discard-empty already did not remove the property if the value is a space, the bug occurred in combination with postcss-normalize-whitespace (first postcss-normalize-whitespace would trim the space, reducing the value to empty, which would cause discard-empty to remove it).

Reply:
I've added a property with a space value to the test just as a reminder, although that case already passed before.